extras/VisitorFamily.js

```diff
@@ -0,0 +1,170 @@
+'use strict';
+
+// --------------------------------------------------------------------
+// Imports
+// --------------------------------------------------------------------
+
+var assert = require('../src/common').assert;
+
+// --------------------------------------------------------------------
+// Private stuff
+// --------------------------------------------------------------------
+
+// Helpers
+
+function getProp(name, thing, fn) {
+  return fn(thing[name]);
+}
+
+function mapProp(name, thing, fn) {
+  return thing[name].map(fn);
```

Comment:
I don't exactly remember why, but I had to add a check here to get it running with my (potentially) nullable properties (if an AST can have child nodes as property or the property is null):
```javascript
if (!(thing[name] instanceof Array)) {
  return null;
}
```